build.gradle

```
@@ -59,9 +67,10 @@ sonarqube {
     }
 }
 
-jacocoTestReport {
-    reports {
-        xml.enabled true
+spotless {
+    java {
+        target "src/**/*.java"
+        googleJavaFormat("1.13.0").aosp()
+        importOrder "", "javax", "java", "\\#"
```

Let's enforce the newline requirement:
```suggestion
        importOrder "", "javax", "java", "\\#"
        endWithNewline()        
```

---

added `endWithNewLine()`